python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## 1.4.0
+
+### Feature:
+- Support for SFTP as file location [#1481](https://github.com/astronomer/astro-sdk/pull/1481)
```

Comment:
here we have PR

Reply:
For astro-sdk if you look at most of them its issues rather than PR: https://raw.githubusercontent.com/astronomer/astro-sdk/main/python-sdk/docs/CHANGELOG.md We should have some consistency there.